Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1177,12 +1177,15 @@ extension BrowserViewController: TabManagerDelegate {
             }
         }
 
-        toolbar?.updateBackStatus(selected?.canGoBack ?? false)
-        toolbar?.updateForwardStatus(selected?.canGoForward ?? false)
-        toolbar?.updateReloadStatus(selected?.loading ?? false)
+        if let tab = tabManager.selectedTab,
+               webView = tab.webView {
+            updateNavigationToolbarStates(tab, webView: webView)
```

Comment:
This updates `urlBar.currentURL`. You can remove the redundant line above (line 1173) that does the same thing.